src/settings/select-setting/select-setting.ts

```diff
@@ -135,9 +135,7 @@ export class UIPSelectSetting extends UIPSetting {
   }
 
   protected setValue(value: string): void {
-    this.removeEventListener('change', this._onChange);
```

Comment:
Don't we need to prevent change handling when we do this operation?

Reply:
thanks! fixed :)